src/main/java/org/dinosaur/foodbowl/domain/comment/dto/CommentResponse.java

```diff
@@ -0,0 +1,25 @@
+package org.dinosaur.foodbowl.domain.comment.dto;
+
+import java.time.LocalDateTime;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.dinosaur.foodbowl.domain.comment.entity.Comment;
+
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class CommentResponse {
+
+    private Long id;
```

Comment:
```suggestion
    private Long commentId;
```
의미를 명시해주는 것도 좋을 것 같아요~~